recipe/meta.yaml

```diff
@@ -23,7 +24,7 @@ requirements:
     - numpy
     - packaging
     - pandas
-    - python
+    - python >=3.6
     - xarray
 
 test:
```

Comment:
Is the pygmt package tested? I don't see any tests in the CI outputs.

Reply:
The imports are tested and that is sufficient here.

Reply:
If there is no import that would ensure the ctype code is executed, you can add an import that ensured that, or you can add a simple test file that ensures gmt function calls can be made.

Reply:
We could run `pygmt.show_versions()` I suppose to check that things are loaded?

Reply:
The issue is that the test isn't run on windows anymore. I think you will need to check the installation on a Windows computer after this is merged to be sure.

Reply:
Yes, I'll keep this PR on hold until tomorrow perhaps (when I can access a Windows computer at uni).